lib/index.js

```diff
@@ -11,12 +11,13 @@ if (process.env.NODE_ENV !== 'production') {
 }
 
 module.exports = babelHelpers._extends({}, configure, {
-  DropdownList: require('./DropdownList'),
-  Combobox: require('./Combobox'),
   Calendar: require('./Calendar'),
+  Combobox: require('./Combobox'),
   DateTimePicker: require('./DateTimePicker'),
-  NumberPicker: require('./NumberPicker'),
+  DropdownList: require('./DropdownList'),
+  ListMultiGroupable: require('./ListMultiGroupable'),
   Multiselect: require('./Multiselect'),
+  NumberPicker: require('./NumberPicker'),
```

Comment:
n/b: Same as above w/ whitespace, if we are going to hack on 3rd party modules we should only do the minimum required changes.